### ISSUE TITLE
Cannot use 4.5.1 Class Library in ASP.NET 5 beta 7

### ISSUE THREAD

**Issue by seanmerron** [aspnet/Home#961]
Hello,

I created a new ASP.NET 5 Beta 7 web project and then add a dll reference to a .NET 4.5.1 Class Library (Non-DNX). Intellisense let's me use the library but at compile time I get "The type or namespace name 'MyLibrary' could not be found (are you missing a using directive or an assembly reference?)."

I thought this feature was available but appears to be not be working. On a related note, will we ever be able to "Add Existing Project" to src for a .NET 4.5.1 Class Library csproj and add reference that way for code shared between multiple apps? Or will the dll/nuget reference be the only support?

Thanks for all of the great work you all are doing!

**Comment by Kagamine** [comment]
what's in your project.json

**Comment by Kagamine** [comment]
May be, you should remove the dnxcore50 from the frameworks in your project.json

**Comment by seanmerron** [comment]
Doh! That makes complete sense and did the trick! Once I removed core, it compiled fine. I guess the little intellisense warning icons for non-core code don't come up in these cases (Maybe a future improvement).

I also tested leaving dnxcore50 in project.json and then wrapping the usage of the 4.5.1 library code in #if DNX451 which also worked.

The compiler error in this scenario also helps showing the project where the error exists as WebApplication3.DNX Core 5.0 which helps me hint at being a core50 error. Thanks again!
